minpy/core.py

```diff
@@ -130,16 +130,40 @@ class MinpyWrapperError(TypeError):
     pass
 
 
-def numpy_to_minpy(var):
+def _numpy_to_minpy(var):
     """ Convert a numpy array to minpy array """
     return array.Value.wrap(var)
 
 
-def minpy_to_numpy(var):
+def _minpy_to_numpy(var):
```

Comment:
Hmm, I see. We just keep it right now. I still don't like the current way of convert from/to numpy to minpy array. This is a core problem in our ways of interface, and we could try fix it in the long term.